CleverSDK/Classes/CLVLoginButton.m

```diff
@@ -26,12 +24,8 @@ @interface CLVLoginButton ()
 
 @implementation CLVLoginButton
 
-+ (CLVLoginButton *)buttonInViewController:(UIViewController *)viewController
-                            withDistrictId:(NSString *)districtId
-                             successHander:(void (^)(NSString *))successHandler
-                            failureHandler:(void (^)(NSString *))failureHandler {
++ (CLVLoginButton *)buttonInViewController:(UIViewController *)viewController {
```

Comment:
Earlier, we were passing in the parent ViewController because we were presenting `CLVOAuthWebViewController` on top of the parent. Since that no longer happens from the button's code, we don't need to pass in the parent.